api/converter/to_bytes.go

```diff
@@ -26,15 +28,20 @@ import (
 
 // ObjectToBytes converts the given object to byte array.
 func ObjectToBytes(obj *json.Object) ([]byte, error) {
-	bytes, err := proto.Marshal(toJSONElement(obj))
+	elem, err := toJSONElement(obj)
```

Comment:
To distinguish protobuf message from other objects, the prefix `pbXXX` is used. How about changing this to `pbElem`?

```suggestion
	pbElem, err := toJSONElement(obj)
```